src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Could we not simply ```e.originalEvent = undefined;```? after doing the extend? That would explicitly sever the connection to the donor event's ```originalEvent```.

OTOH, considering all the changes involved, it may simply be easier to revert https://github.com/jquery/jquery/commit/a873558436383beea7a05fd07db7070a30420100